src/test/java/org/cactoos/scalar/PropertiesOfTest.java

```diff
@@ -122,7 +123,7 @@ public void sensesChangesInMap() throws Exception {
         new Assertion<>(
             "Must sense the changes in the underlying map",
             props.value().size(),
-            Matchers.not(Matchers.equalTo(props.value().size()))
+            new IsNot<>(new IsEqual<>(props.value().size()))
```

Comment:
@marceloamadeu why not use the `HasSize` matcher as proposed?

Reply:
I couldn't use `HasSize` ... Any idea how I can fix the error:

`new IsNot<>(new HasSize(props.value().size()))`

```
[ERROR]  cannot infer type arguments for org.llorllale.cactoos.matchers.Assertion<>                                                                               
[ERROR]  reason: inference variable T has incompatible bounds
[ERROR]  equality constraints: T,java.lang.Iterable<?>
[ERROR]  lower bounds: java.lang.Integer
```
Tks

Reply:
@marceloamadeu HasSize takes a collection or an iterable, not the result of calling `size()`, check its signature.

Also can you update your comment to specify who (me) you are talking to? It will have impact on the quality assessment done on my work at the end of my review.

Reply:
@marceloamadeu or sorry, it's not exactly true, the first argument of the assertion should be a collection or iterable, not the argument of HasSize

Reply:
@victornoel Sorry ... I made the correction.

> Also can you update your comment to specify who (me) you are talking to? It will have impact on the quality assessment done on my work at the end of my review.

Reply:
@marceloamadeu thx